assets/js/glossary.js

```diff
@@ -13,6 +13,7 @@ $gp.glossary = (
 				}
 				$gp.glossary.table = table;
 				$gp.glossary.install_hooks();
+				$gp.glossary.tablesorter();
```

Comment:
Shouldn't we make sorting also available to users you cannot edit entries? I'd move it right below `$gp.init();`.

Reply:
Agree, doing it.